lib/imgix/rails/view_helper.rb

```diff
@@ -8,36 +8,11 @@ module Rails
     module ViewHelper
       include UrlHelper
 
-      def ix_image_tag(*args)
-        case args.size
-        when 1
-          Imgix::Rails::ImageTag.new(args[0]).render
-        when 2
-          source = args[0]
-          options = args[1]
-          tag_options = options[:tag_options] || {}
-          url_params = options[:url_params] || {}
-          widths = options[:widths] || []
-
-          if options.except(:tag_options, :url_params, :widths).length > 0
-            ActiveSupport::Deprecation.warn('options Hash is deprecated; use :tag_options, :url_params, :widths instead.')
-            tag_options = options.slice!(*Imgix::Rails::Tag.available_parameters).except(:widths)
-            url_params = options.slice(*Imgix::Rails::Tag.available_parameters).except(:widths)
-          end
-
-          Imgix::Rails::ImageTag.new(source, tag_options: tag_options, url_params: url_params, widths: widths).render
-        end
+      def ix_image_tag(source, tag_options: {}, url_params: {}, widths: [])
+        Imgix::Rails::ImageTag.new(source, tag_options: tag_options, url_params: url_params, widths: widths).render
       end
 
-      def ix_picture_tag(source, picture_tag_options: {}, imgix_default_options: {}, breakpoints:, tag_options: {}, url_params: {})
```

Comment:
Same as above--can we pull all of the code that's just removing deprecated things out into a separate PR? We can bundle the two so they're merged and released together, but I think they should be kept separate.

Reply:
I will move this into a separate PR.